perf/lua/1mops_write.lua

```diff
@@ -0,0 +1,294 @@
+-- Usage: taskset 0xf0 tarantool relay-1mops.lua [options]
+--
+-- Options can be used with '-' or '--', followed by the value if it's not
+-- a boolean option. The options list with default values:
+--   engine <string, 'memtx'> - select engine type
+--   fibers <number, 50> - number of fibers to run simultaneously
+--   index <string, 'HASH'> - select index type
+--   nodes <number, 1> - number of nodes to test one-way replication
+--   nohint <boolean> - to turn the hints off
+--   ops <number, 10000000> - total amount of replaces to be performed
+--   sync <boolean> - to turn the synchro replication on
+--   transaction <number, 150> - number of replaces in one transaction
+--   warmup <number, 10> - percent of ops to skip before measurement
+--
+-- Being run without options tests the HASH index of memtx engine by
+-- running of 50 fibers that replace 150 integers using each as a primary
+-- key. Refer to the 'fiber_load' function below to change the workload.
+
+local err, popen = pcall(require,'popen')
+if err ~= nil then
+    popen = {}
+end
+local clock = require('clock')
+local fiber = require('fiber')
+local math = require('math')
+local json = require('json')
+
+-- working dir is cleared/created for every run, set test_dir
+-- during the test a subdir will be created for each replica
+local function getarg(s, f)
```

Comment:
It would be nice to tweak the benchmark setup via environment variables instead of the CLI options to run it within container.

Reply:
Hmm... I suppose running it in any form of virtual environment will hurt the stability of the test.
Nevertheless - I can handle it, perhaps along with rewrite the arguments parsing through the internal.argparse.